consensus/pacemaker/module.go

```diff
@@ -243,6 +265,82 @@ func (m *pacemaker) NewHeight() {
 		)
 }
 
+// This is called each time the system wants a delayed signal to build a block
```

Comment:
Let's follow proper godoc comments: https://tip.golang.org/doc/comment

Unrelated request: If you are finding that I'm making requests that you didn't read about in https://github.com/pokt-network/pocket/blob/main/docs/development/CODE_REVIEW_GUIDELINES.md?
```suggestion
// RegisterMinBlockTimeDelay should be called when a delay should be introduced into proposing a new block
```